Sprint-2/errors/2.js

```diff
@@ -3,8 +3,17 @@
 
 // this function should square any number but instead we're going to get an error
 
-function square(3) {
+// function square(3) {
+//     return num * num;
+// }
+// there is a value input in the function, thus it throw a SyntaxError, in Javascript a value should not be put in the function definition.
```

Comment:
What kind of error exactly? Can you link to MDN to show the rule about function parameters?